scan/scan.go

```diff
@@ -56,12 +63,25 @@ func NewScanner(opts options.Options, cfg config.Config) (Scanner, error) {
 		}
 	}
 
-	// load up alternative config if possible, if not use manager's config
+	// load up alternative config if possible, if not use default/specified config. Will merge if MergeRepoConfig is true
 	if opts.RepoConfigPath != "" && !opts.NoGit {
-		cfg, err = config.LoadRepoConfig(repo, opts.RepoConfigPath)
+		repoCfg, err := config.LoadRepoConfig(repo, opts.RepoConfigPath)
+		if opts.MergeRepoConfig && err == nil {
```

Comment:
I would prefer 
```golang
	if opts.RepoConfigPath != "" && !opts.NoGit {
		repoCfg, err := config.LoadRepoConfig(repo, opts.RepoConfigPath)
		if err != nil {
			return nil, err
		}
		if opts.MergeRepoConfig {
			cfg = cfg.Merge(repoCfg)
		} else {
			cfg = repoCfg
		}
	}
```

Reply:
Totally agree, much more readable this way. Not sure why I wrote it the way I did  🤷‍♂️.